sdkproject/Assets/Mapbox/Unity/DataContainers/ElevationModificationOptions.cs

```diff
@@ -1,11 +1,16 @@
 ﻿namespace Mapbox.Unity.Map
 {
 	using System;
+	using UnityEngine;
+
 	[Serializable]
 	public class ElevationModificationOptions
 	{
+		[Tooltip("Number of samples to use for terrain mesh.")]
```

Comment:
What does it mean to use X number of samples? Can we add more context here?

Reply:
I think this might be better described as 'resolution' or 'mesh resolution'. The mesh is generated by sampling the tile image 'x' number of times at even intervals.

So '2' would mean a 2x2 grid of vertices per tile, and '10' would mean a 10x10 grid.

Reply:
So " Mesh resolution of terrain, results in n x n grid " . I am so bad at defining this ;)